### DISCUSSION TITLE
kairosdb datasource query not working with latest grafana version

### DISCUSSION THREAD

**Discussion by mdvaliyullah** [grafana/grafana#52924] [Data Sources]
kairosdb datasource query not working with latest version of grafana.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
